features/partnerhistorie/peildatum-periode-filtering.feature

```diff
@@ -0,0 +1,258 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Selecteer het/de correcte (historische) huwelijken geregistreerde partnerschappen.
+
+Rule: Bij vragen van partnerhistorie op peildatum wordt de partner geleverd waarbij de Datum huwelijkssluiting/aangaan geregistreerd partnerschap op of voor de peildatum ligt én de eventuele Datum ontbinding huwelijk/geregistreerd partnerschap na de peildatum.
+
+    Abstract Scenario: Peildatum in actuele huwelijk/partnerschap
+      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende huwelijken/partnerschappen in de registratie
+      | Categorie | Burgerservicenummer (01.20) | Datum aangaan (06.10) | Datum ontbinding (07.10) | 
+      | 5         | 555550002                   | 20171103              |                          | 
+      | 55        | 555550003                   | 20101022              | 20140315                 | 
+      Als partnerhistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | burgerservicenummer             |
+      | peildatum           | 20181017                        |
+      Dan heeft de response een partnerhistorie met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550002 |
+
+    Scenario: Peildatum in ontbonden huwelijk/partnerschap
+      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende huwelijken/partnerschappen in de registratie
+      | Categorie | Burgerservicenummer (01.20) | Datum aangaan (06.10) | Datum ontbinding (07.10) | 
+      | 5         | 555550002                   | 20171103              |                          | 
+      | 55        | 555550003                   | 20101022              | 20140315                 | 
+      Als partnerhistorie wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | burgerservicenummer             |
+      | peildatum           | 20120229                        |
+      Dan heeft de response een partnerhistorie met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550003 |
+
```

Comment:
moet hier dan niet ook de partner:
    | Categorie | Burgerservicenummer (01.20) | Datum aangaan (06.10) | Datum ontbinding (07.10) | 
      | 5         | 555550002                   | 20171103              |                          | 
worden weggelaten?

Misschien is het beter om eerst de opmerkingen van Melvin overal toe te passen, en dat wij dan weer gaan reviewen?